src/domUpdates.js

```diff
@@ -137,6 +142,20 @@ const viewSavedRecipes = (user, ingredientsData) => {
   let deleteBtn = document.querySelectorAll('.delete-btn')
   addDelete(deleteBtn)
   selectRecipe()
+  let groceryIngredients = Object.keys(groceryList)
+  displayGroceryList(groceryList, groceryIngredients)
+  displayGroceryCost(totalGrocCost)
+}
+
+const displayGroceryList = (groceryList, groceryIngredients) => {
+  const ingredients = groceryIngredients.map((ingredient) => {
+    return `¤ ${ingredient}: ${groceryList[ingredient].amount} ${groceryList[ingredient].unit} \n ‣ $${groceryList[ingredient].estimatedCostInCents / 100}`
+    })
```

Comment:
if estimatedCostInCents is the cost per unit, we might need to multiply estimatedCostInCents by amount before diving by 100

Reply:
ooh I forgot about that, great catch!

Reply:
Oh snap! You're so right. I pushed the changes. Good eye.